resources/public/api/conf/1.0/application.yaml

```diff
@@ -42,15 +42,26 @@ paths:
             application/vnd.hmrc.1.0+json:
               schema:
                 $ref: '#/components/schemas/ExciseMovement'
-            application/vnd.hmrc.1.0+xml:
-              schema:
-                $ref: '#/components/schemas/ExciseMovement'
         '400':
           description: The submission of draft movement message has failed validation
+          content:
```

Comment:
this may need a discussion and we may want to return a 503 too, like the EIS does.

Reply:
yeah I think you are right we probably want to add that here.